addons/stock/models/stock_inventory.py

```diff
@@ -179,21 +192,36 @@ def _get_inventory_lines_values(self):
         locations = self.env['stock.location'].search([('id', 'child_of', [self.location_id.id])])
         domain = ' location_id in %s'
         args = (tuple(locations.ids),)
+        vals = []
+        Product = self.env['product.product']
```

Comment:
![](http://www.troll.me/images/another-boromir-meme/one-does-not-simply-make-soup-thumb.jpg)